src/CommunityToolkit.Maui/Converters/ICommunityToolkitValueConverter.shared.cs

```diff
@@ -5,6 +5,16 @@ namespace CommunityToolkit.Maui.Converters;
 /// <inheritdoc />
 public interface ICommunityToolkitValueConverter : IValueConverter
 {
+	/// <summary>
+	/// Type Converting From
```

Comment:
Perhaps something like

```suggestion
	/// Gets the <see cref="Type" /> this converter expects to <see cref="Convert" /> from or <see cref="ConvertBack" /> to.
```